lib/r/source.dart

```diff
@@ -190,6 +199,17 @@ Future<void> rSource(BuildContext context, WidgetRef ref, String script) async {
   code = code.replaceAll('STEM', stem ? 'TRUE' : 'FALSE');
   code = code.replaceAll('PUNCTUATION', punctuation ? 'TRUE' : 'FALSE');
   code = code.replaceAll('STOPWORD', stopword ? 'TRUE' : 'FALSE');
+  if (!clusterReScale) {
```

Comment:
I don't want to see R code here - it should be in the R buld script, and choose one approach or the other within the R script.

Reply:
Why was clusterRescale in r/source.dart placed under the WORDCLOUD transforms? Seems like a very odd place to put it. I moved it to be with CLUSTER below.